### DISCUSSION TITLE
2024-02-29 Evaluator and Performance Office Hours

### DISCUSSION THREAD

**Discussion by myitcv** [cue-lang/cue#2862] [Community Calls]
As a follow up to #2850 and the nexts steps outlined in #2857, we are arranging an Evaluator and Performance Office Hours on Thursday 29 Feb, 2024 at 1600 UTC.

Please reply with any questions/topics you would like to see covered. This is an entirely open session designed largely to cover the questions and topics brought up in #2857. 

### Meeting details

https://meet.google.com/zkh-dfip-jqc

As usual, this session will be recorded for those unable to join us live.

### CUE Community shared calendar

If you want to stay up to date with future community calls, please [subscribe to the shared calendar](https://cuelang.org/s/community-calendar).

### Recording

https://youtu.be/Jr4K0rQ342Q

### Slides/materials

https://docs.google.com/presentation/d/1eqdETBPmf06zXu1rYOYhji_o3xltopkkLps9pCYonuE/edit?usp=sharing

### Auto-generated transcript

https://docs.google.com/document/d/1sdZtlKjL2hej5pGqnIb4D4dse_oVX0oYlpKqNfbodO0/edit?usp=sharing

**Comment by infogulch** [comment]
Purely out of curiosity I'd like to hear an overview about how the old evaluator worked. And even more, how the new evaluator is being designed. Are you using an existing unification-based logic evaluator theory for inspiration, e.g. prolog/datalog? Or making something custom to cue?

**Reply by myitcv** [reply]
@infogulch - thanks for the question, very happy to cover this briefly later today. I say briefly because this answer could take the entire hour! Happy to follow up afterwards with further links, pointers etc too.

**Comment by myitcv** [comment]
Description above updated with links to the recording, slides and transcript.
